### PR TITLE
Added right mouse button click support

### PR DESCRIPTION
**Pull request by harshlancer**
I've tried to add the right mouse button and updated the HTML file as well as the README file.
Please tell me if I can improve.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ The following actions are supported:
 - Seek controls (<< and >>)
 - Mouse cursor movement
 - Left-mouse-button click using tapping
+- Rigth-mouse-button double click using double tapping
```
Comment (shubham1172) [inline README.md:15]: nit: Spelling of `right`. Should it read instead `Right-mouse-button click using double tapping`? I believe the `double` is extra.

File: static/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (shubham1172) [inline static/.vscode/settings.json:2]: Was this file added by mistake?

File: static/index.html [hunk 3]
```diff
@@ -41,8 +41,8 @@ <h1 class="title">Remote Play</h1>
       </button>
       <div class="touchpad" id="touchpad">
         <p>
-          Use this area as a touchpad.<br />Supported gestures:<br />track and
-          tap (Left Mouse Button).
+          Use this area as a touchpad.<br />Supported gestures:<br />Track and
+          Tap (Left Mouse Button).<br/>Double Tap (Right Mouse Button).
```
Comment (shubham1172) [inline static/index.html:45]: Let's break this into multiple lines for readability. Also I don't think we should have a line break before `Double tap`.
```html
Use this area as a touchpad.<br/>
Supported gestures:<br/>
Track, tap (left mouse button), and double tap (right mouse button).
```

File: static/index.js [hunk 4]
```diff
@@ -37,4 +37,10 @@ document.addEventListener('DOMContentLoaded', (event) => {
     // left mouse button click
     ws.send(JSON.stringify({ type: 'tap' }));
   });
+
+  hammer.on('double tap', (ev) => {
```
[the remark this answers is not in the document]
Comment (shubham1172) [reply static/index.js:41]: Also, you are trying to send a web socket event here, but you also need logic to receive that event and translate it to right mouse button click. As shared earlier, https://github.com/shubham1172/remote-play/blob/main/main.py holds that code. See line 30 for how single tap is handled.

File: README.md (hunk 1 above)
Comment (harshlancer) [reply README.md:15]: Ok I'll surely fix that , thank-you very much for your guidance.

File: static/.vscode/settings.json (hunk 2 above)
Comment (harshlancer) [reply static/.vscode/settings.json:2]: Sorry it definitely was my mistake

File: static/index.html (hunk 3 above)
Comment (harshlancer) [reply static/index.html:45]: Ok I'll do that..

File: static/index.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (harshlancer) [reply static/index.js:41]: I've resolved the issues mentioned above but here I think I lack fundamentals it'd be very nice of you if you could point me to some sources where I can find the right mentorship about this topic.
Thank you for your guidance.

[the remark this answers is not in the document]
Comment (shubham1172) [reply static/index.js:41]: The three main things involved here are 
1. https://hammerjs.github.io/getting-started/ - on client side to handle events, based on user input
1. https://flask.palletsprojects.com/en/2.0.x/ - web server that listens for hammer events 
1. https://pyautogui.readthedocs.io/en/latest/ - server side library to invoke mouse and keyboard actions

You can start reading the official docs and try reading and running the existing code, getting a feel of how things work.

**Review by harshlancer — COMMENTED** [review]
I wasn't able to add a new click button in the index.js file but I've updated other files.

Screenshot 2022-02-10 14 15 38
Screenshot 2022-02-10 14 15 23
Screenshot 2022-02-10 14 15 00
es.

**Comment by shubham1172** [comment]
> I wasn't able to add a new click button in the index.js file but I've updated other files.

No worries @harshlancer, and thanks for trying! Once you pick up the other parts, feel free to contribute back to this repo.

**Pull request closed without merging**